keras/ops/nn.py

```diff
@@ -1840,3 +1840,57 @@ def ctc_loss(target, output, target_length, output_length, mask_index=0):
     return backend.nn.ctc_loss(
         target, output, target_length, output_length, mask_index
     )
+
+
+class Normalize(Operation):
+    def __init__(self, order=2.0, axis=1, eps=1e-12):
+        super().__init__()
+        self.p = order
+        self.axis = axis
+        self.eps = eps
+
+    def compute_output_spec(self, x):
+        return KerasTensor(shape=x.shape)
+
+    def call(self, x):
+        return _normalize(x, self.order, self.axis, self.eps)
+
+
+@keras_export(
+    [
+        "keras.ops.normalize",
+        "keras.ops.nn.normalize",
+    ]
+)
+def normalize(x, order=2.0, axis=1, eps=1e-12):
```

Comment:
Use `axis=-1, order=2` for backwards compat.

`eps=1e-12` is not going to work in float32 or float16. It seems to assume float64. Just omit it from the signature and use `config.epsilon()` in the code.